index.html

```diff
@@ -33,8 +150,63 @@ <h2 class="blue-text">LET'S CONNECT</h2>
       <li><a href="#"><img src="./img/github.png" alt="Github"></a></li>
       <li><a href="#"><img src="./img/angel-list.png" alt="Angel-list"></a></li>
     </ul>
+    <button class="big-button" type="submit" name="submit">Get My Resume</button>
+    <ul class="frame flex-button">
+      <li>
+        <h2>Languages</h2>
+      </li>
+      <li><img src="./img/Vector.png" alt="lang"></li>
+    </ul>
+
+    <div class="buttons">
+      <div class="button-fin">
+        <img src="./img/js.svg" alt="js-button">
+        <p>JavaScript</p>
+      </div>
+    </div>
+
+    <div class="buttons">
+      <div class="button-fin">
+        <img src="./img/html.svg" alt="html-button">
+        <p>HTML</p>
+      </div>
+    </div>
+
+    <div class="buttons">
+      <div class="button-fin">
+        <img src="./img/css.svg" alt="css-button">
+        <p>CSS</p>
+      </div>
+    </div>
+    <ul class="frame">
+      <li>
+        <h2>Framworks</h2>
+      </li>
+      <li>></li>
+    </ul>
+    <ul class="frame">
+      <li>
+        <h2>Skills</h2>
+      </li>
+      <li>></li>
+    </ul>
 
   </section>
+
+  <footer>
+    <h2 class="contact-me">Contact Me</h2>
+    <p class="white-text">If you have an application you <br>
+      are interested in developing, a <br>
+      feature that you need built or a <br> project that needs coding. I’d <br>
+      love to help with it </p>
+
+    <input class="name-area" type="text" placeholder="Full name"><br>
+    <input class="mail-area" type="text" placeholder="Email adress"><br>
+    <textarea name="User_message" id="msg" placeholder="Write your message here" cols="30" rows="10"></textarea><br>
+    <button class="end-button" type="submit" name="submit">Get It Touch</button>
```

Comment:
This button is on the left side of the form and there is a background image on the right side. Please follow the design guidelines and fix the issue. Kindly see the below screenshot for reference.
![button](https://user-images.githubusercontent.com/35504149/130312170-6c081585-2139-44a8-aae9-e851e5a4655f.png)

Reply:
Thank you for your precious reviews, I'm working on them as soon as possible.

Reply:
In my browser it is centered, I checked many times, what can I do?

Reply:
@SajjadAhmad In my browser it is centered, I checked many times, what can I do please?

Reply:
Fixed.